src/components/AvailableImagesTile.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {
+  Card,
+  CardTitle,
+  CardBody,
+  Button,
+  Spinner,
+} from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+
+const AvailableImageTileBase = ({ children }) => {
+  return (
+    <Card>
+      <CardTitle>Available images</CardTitle>
+      {children}
+    </Card>
+  );
+};
+
+AvailableImageTileBase.propTypes = {
+  children: PropTypes.children,
+};
+
+const AvailableImageTile = () => {
+  const { status, data } = useSelector(({ imagesReducer }) => ({
+    status: imagesReducer?.status || 'PENDING',
+    data: imagesReducer?.data || null,
+    error: imagesReducer?.error || null,
+  }));
+
+  let content = null;
+  if (status === 'PENDING') {
+    content = (
+      <>
+        <CardBody
+          isFilled
+          style={{
+            display: 'flex',
+            justifyContent: 'center',
+            alignItems: 'center',
+          }}
+        >
+          <Spinner />;
+        </CardBody>
+      </>
+    );
+  }
+  if (status === 'DONE' && !data.success) {
+    content = <CardBody>{data.payload}</CardBody>;
+  }
+  if (status === 'DONE' && data.success) {
+    content = (
+      <>
```

Comment:
I don't really like this, it's easy to overlook and you have to think what it actually does. Much cleaner approach is with `Fragment` component from React.